scripts/normalize

```diff
@@ -75,12 +75,17 @@ def read_and_write(input_file:, output_file:)
   yield(doc) if block_given?
 
   File.open(output_file, 'w') do |f|
-    f.write doc.to_xhtml(indent:2)
+    f.write doc.to_xhtml(indent: 2)
   end
 end
 
 ARGV.each do |input_file|
-  output_file = input_file.gsub(/(\.\w+)\z/, '.normalized\1')
+  output_file =
+    if File.extname(input_file) != ''
+      input_file.gsub(/(\.\w+)\z/, '.normalized\1')
+    else
+      "#{input_file}.normalized"
+    end
```

Comment:
this is why `normalize` was making changes in place to the spec files -- there was no extension on the temp files in rspec, so no `gsub` was happening and `output_file` was the same as `input_file`.